daemon/execdriver/utils.go

```diff
@@ -59,6 +64,8 @@ func GetAllCapabilities() []string {
 	return output
 }
 
+// TweakCapabilities can tweak capabilities by adding or dropping capabilities
```

Comment:
I'd add a mention that capabilities here refers to capabilities provided by the kernel, and an example (maybe even a link to a lwn article or something?).